src/2982R0_std_quantity_as_a_numeric_type.md

```diff
@@ -991,18 +993,136 @@ The above program will produce the following types for acceleration quantities
 
 ## Scaled units
 
+Another very common operation is to multiply an existing unit by a factor, creating a new, scaled unit.  For
+example, the unit _foot_ can be multiplied by 3, producing the unit _yard_.
+
+The process also works in reverse: the ratio between any two units of the same dimension is a well-defined
+number.  For example, the ratio between one foot and one inch is 12.
+
 ### Unit magnitudes
 
-TBD
+In principle, this scaling factor can be any positive real number.  In mp-units and Au, we have used the term
+"magnitude" to refer to this scaling factor.  (This should not be confused with the logarithmic "magnitude"
+unit commonly used in astronomy.)
```

Comment:
What about these magnitudes?
- https://en.wikipedia.org/wiki/Magnitude_(mathematics)
- https://www.electropedia.org/iev/iev.nsf/display?openform&ievref=102-03-23

I'm exploring just using "vector" for everything that's a vector,
and relying on the abstraction levels to make it clear from context.
Even if I have something like `pos.x`,
where `pos` is of type `cartesian::vector2d`,
and `pos.x` is of type `qty::vector` (a.k.a. `mp_units::quantity` with a scalar number).